Arm/RoverArmSystem.hpp

```diff
@@ -0,0 +1,77 @@
+#pragma once
+#include "utility/units.hpp"
+#include "joint.hpp"
+#include "wrist_joint.hpp"
+
+namespace sjsu::arm
+{
+class RoverArmSystem
+{
+ private:
+  // isOperational defines if the arm should be allowed to move or if it should
+  // be in its resting 'off' position.
+  bool isOperational;
+
+  sjsu::arm::Joint & Rotunda;
+  sjsu::arm::Joint & Shoulder;
+  sjsu::arm::Joint & Elbow;
+  sjsu::arm::WristJoint & Wrist;
+
+  // The target angle for each of the joints
+  units::angle::degree_t shoulder_pos;
+  units::angle::degree_t elbow_pos;
+  units::angle::degree_t rotunda_pos;
+  units::angle::degree_t wrist_roll_pos;
+  units::angle::degree_t wrist_pitch_pos;
+
+ public:
+  RoverArmSystem(sjsu::arm::Joint rotunda,
+                 sjsu::arm::Joint shoulder,
+                 sjsu::arm::Joint elbow,
+                 sjsu::arm::WristJoint wrist)
```

Comment:
If you want to take these as a parameter then take them as a reference. But you don't need to since all of this can be kept within the class. You just need a way to distinguish between each joint in some fashion in this class.